code/modules/tgui/modules/atmos_control.dm

```diff
@@ -28,7 +30,15 @@
 	)
 
 /datum/ui_module/atmos_control/ui_data(mob/user)
+	var/list/alarms_on_net = GLOB.air_alarms
+	// default value means main station atmos control
+	if(parent_area_type)
+		alarms_on_net = list()
+		for(var/obj/machinery/alarm/Alarm in GLOB.air_alarms)
```

Comment:
Same here, this var name isn't consistent with our style

Reply:
same here, I assume?